source/agora/utils/Test.d

```diff
@@ -740,15 +741,21 @@ public struct TxBuilder
 
 /// Returns:
 ///   A range of `TxBuilder`s which reference each Payment output of
-///   the Genesis block
-public auto genesisSpendable () @safe
+///   the input block
+public auto spendable (in Block b) @safe
```

Comment:
For the name of the parameter `b`, `block` would be more readable. The function is very short though.